File: Android/tanika11_TanikaGulati_2024ec1060_2/GettingStarted/Print_Z.cpp
#include<iostream>
using namespace std;
int main(){
    //write your code here
    for(int i=1;i<6;i++)
    {
        if(i>1&&i<5)
        {
            for(int j=0;j<5-i;j++)
            {
                cout<<" ";
            }
            cout<<"*"<<endl;
        }
        else
        {
            for(int j=0;j<5;j++)
            cout<<"*";
            cout<<endl;
        }
    }
}

File: IOT/shakap12_ShashwatKapoor_1923ec1177_3/GettingStarted/PrintZ.java
import java.util.*;

public class Main {

    public static void main(String[] args) {
        // Write your code here
         int k=4;
        for(int x=0;x<5;x++){
            if(k==4 || k==0){
                for(int y=0;y<5;y++)
                System.out.print("*");
            }
            else
            for(int y=0;y<5;y++){
                if(y==k)
                System.out.print("*");
                else
                System.out.print(" ");
            }
            System.out.println();
                k--;
            }

    }
}